docs/user/TerratestDockerUsage.md

```diff
@@ -0,0 +1,98 @@
+# Using the Terratest Docker Container
+
+Use the Terratest Docker container to run the suite of Terratest Go tests. For more information on Terratest, follow the [Documentation](https://terratest.gruntwork.io/docs/) page. The Terratest Docker image is used by the [Github Workflow](../../.github/workflows/default_plan_unit_tests.yml) as a required check before merging changes.
+
+## Prereqs
+
+- Docker [installed on your workstation](../../README.md#docker).
+
+## Preparation
+
+### Docker image
+
+Run the following command to create the `viya4-iac-azure-terratest` Docker image using the provided [Dockerfile.terratest](../../Dockerfile.terratest)
+
+```bash
+docker build -t viya4-iac-azure-terratest -f Dockerfile.terratest .
+```
+
+The Docker image `viya4-iac-azure-terratest` will contain Terraform and Go executables, as well as the required Go modules. The Docker entrypoint for the image is `go test` and accepts several optional command line arguments. For more information on command line arguments, please see [Command Line Arguments](#command-line-arguments).
+
```

Comment:
The Docker entrypoint for the image is `go test`, and it accepts several optional command-line arguments. For more information about command-line arguments, see [Command-Line Arguments](#command-line-arguments).